packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -105,6 +107,7 @@ export const publishDescriptorErrorMapper = (
       "eServiceDescriptorNotFound",
       () => HTTP_STATUS_NOT_FOUND
     )
+    .with("notValidDescriptor", () => HTTP_STATUS_BAD_REQUEST)
```

Comment:
This mapper is used in the `publishDescriptor` operation. The error was thrown but not handled.